packages/adapters/rollup.config.mjs

```diff
@@ -5,35 +5,38 @@ import commonjs from "@rollup/plugin-commonjs";
 import json from "@rollup/plugin-json";
 import { readFileSync } from "fs";
 
-const pkg = JSON.parse(readFileSync('package.json', {encoding: 'utf8'}));
-
+const pkg = JSON.parse(readFileSync("package.json", { encoding: "utf8" }));
 
 export default {
-  input: pkg.src || 'src/index.ts',
-  output: [
-    // {
-    //   file: `dist/${pkg.name}.js`,
-    //   format: "umd",
-    //   name: pkg.name,
-    //   sourcemap: true
-    // },
-    {
-      file: `dist/${pkg.name}.es.js`,
-      format: "es",
-      sourcemap: true
-    }
-  ],
-  plugins: [
-    resolve({
-      browser: true
-    }),
-    commonjs(),
-    typescript(),
-//    globals(),
-    // builtins(),
-    babel({
-      exclude: "node_modules/**"
-    }),
-    json()
-  ]
+    external: ["dcmjs"],
```

Comment:
Sorry about the refactor - the externals library is this line here - it also needs the globals, but I don't quite know how to figure out the global name for the other externals.